apis/experience-platform/Identity Service.postman_collection.json

```diff
@@ -1216,148 +704,57 @@
-									"    if (header.value.startsWith(\"{{\") && header.value.endsWith(\"}}\")) {",
-									"        if (!pm.variables.get(header.value.substring(2, header.value.length - 2))) { pm.request.headers.remove(header.key); }",
-									"    }",
-									"});",
-									"",
-									"// Do not send HTTP URL Query Parameters with empty variables, as Postman will send the literal variable name",
-									"pm.request.url.query.remove(q => { ",
-									"    if (q.value.startsWith(\"{{\") && q.value.endsWith(\"}}\")) {",
-									"        return !pm.variables.get(q.value.substring(2, q.value.length - 2));",
-									"    } ",
-									"});",
-									"/** End Adobe-provided Pre-Request Scripts **/"
-								]
-							}
-						}
-					],
-					"_postman_id": "d328792a-4ae2-5dee-8d49-e84b2784058a",
-					"request": {
-						"method": "GET",
-						"header": [
-							{
-								"key": "Accept",
-								"value": "application/json"
-							},
-							{
-								"key": "Authorization",
-								"value": "Bearer {{ACCESS_TOKEN}}",
-								"description": "The access token which can be copied from your Experience Platform integration, prefixed with \"Bearer \".  For more information on how to obtain this value, visit the [authentication tutorial](http://www.adobe.com/go/platform-api-authentication-en).",
-								"type": "string",
-								"enabled": true
-							},
-							{
-								"key": "x-api-key",
-								"value": "{{API_KEY}}",
-								"description": "The API key which can be copied from your Experience Platform integration. For more information on how to obtain this value, visit the [authentication tutorial](http://www.adobe.com/go/platform-api-authentication-en).",
-								"type": "string",
-								"enabled": true
+								"description": "The the all-lowercase name of the sandbox the operation will take place in. If unspecified, falls back to the default production sandbox (usually \"prod\"). See the [sandboxes overview](https://adobe.com/go/sandbox-overview-en) for more information.\n"
 							},
 							{
-								"key": "x-gw-ims-org-id",
-								"value": "{{IMS_ORG}}",
-								"description": "The Organization ID which can be copied from your Experience Platform integration. For more information on how to obtain this value, visit the [authentication tutorial](http://www.adobe.com/go/platform-api-authentication-en).",
-								"type": "string",
-								"enabled": true
+								"key": "Content-Type",
+								"value": "application/json"
 							},
 							{
-								"key": "x-sandbox-name",
-								"value": "{{SANDBOX_NAME}}",
-								"description": "Identifies the Adobe Experience Platform sandbox to use. Default sandbox is 'prod'",
-								"type": "string",
-								"enabled": true
+								"key": "Accept",
+								"value": "application/json"
 							}
 						],
 						"body": {
-							"mode": "formdata",
-							"formdata": []
+							"mode": "raw",
+							"raw": "{{body}}",
```

Comment:
in the description we could leave either an example or point to developer doc on how to specify body content?

Reply:
All collections in this repo follow this pattern of the body being a placeholder, so I'd rather not deviate from under the current scope

Reply:
in terms of usability seams wrong.
can we leave. comment to go for body to https://developer.adobe.com/experience-platform-apis/references/identity-service/ ?